### DISCUSSION TITLE
Are ideal white balance coefficients intensity dependent?

### DISCUSSION THREAD

**Discussion by tlkvstepan** [colour-science/colour#1215] [General]
Hello, 

Thank you for amazing work. I have a question related to white balance. I have following piece of code
```
reference_color_checker = colour.CCS_COLOURCHECKERS['ColorChecker24 - After November 2014']
reference_swatches = colour.XYZ_to_RGB(
        XYZ=colour.xyY_to_XYZ(list(reference_color_checker.data.values())),
        illuminant=reference_color_checker.illuminant,
        colourspace=colour.models.RGB_COLOURSPACE_sRGB,
        apply_cctf_encoding=False,
    )
```
Last six swatches in the color chart correspond to gray patches with different intensity. Therefore, they all should lead 
to same color balancing coefficients. However, they all producing slightly different coefficients:

![Screenshot from 2023-10-23 13-54-02](https://github.com/colour-science/colour/assets/11693251/918dfc25-b00a-421a-9f42-2b05e8c9860b)

What can be the reason for this behavior?

**Comment by thomasmansencal** [comment]
The swatches are not exactly neutral so the computed white balancing factors will change as a function of the swatches you are using:

```python
print(reference_swatches[-6:])

[[ 0.87919095  0.88476747  0.8349529 ]
 [ 0.58443959  0.59212352  0.58458201]
 [ 0.35767777  0.36706043  0.36528718]
 [ 0.19008669  0.19086038  0.1898278 ]
 [ 0.08593528  0.08873843  0.08978779]
 [ 0.03135966  0.03149993  0.03231098]]
```

**Reply by tlkvstepan** [reply]
Thank you for reply! What is the reason? Is it due to non-ideal manufacturing process of the chart?

**Reply by thomasmansencal** [reply]
Yes, producing a truly neutral reflectance across the Visible Spectrum at an affordable price point is hard!

**Reply by tlkvstepan** [reply]
Thank you!
